### PR TITLE
Add blog post for sprint 79

### PR DESCRIPTION
**Pull request by krauselukas**
I will update the graphics, as soon as the UI part is ready :)

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *openbuildservice* ready!

Built with commit 9467651c735bc357e6a7d9f2504faa11322a56b1

https://deploy-preview-260--openbuildservice.netlify.app

**Comment by krauselukas** [comment]
[Communicate the deliverable to the customers](https://trello.com/c/5ikAwGSZ/1222-communicate-the-deliverable-to-the-customers)

File: _posts/development/2020-06-10-events-to-notifications.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Instead of producing new notifications records for every single event, we keep notifications
+related to a certain notifiable(can be a comment or request) up-to-date. It will always reflect
+the current state, till the user confirms the acknowledgement by marking it as read.
+
+Badges are indicating the current state of the associated request. A state change of the request will be
+reflected, by the badge. The avatars of users and groups will tell you, who else is involved since the
+last time you read the notification.
+<img src="/images/posts/sprint-report-79/notification_avatars_and_state.png" alt="Notification avatars and state"/>
+
+For those who wonder where the review wanted notifications remain. This is now unified with the request
```
Comment (hennevogel) [inline _posts/development/2020-06-10-events-to-notifications.md:26]: I' pretty sure no one wonders about this :wink: 

I think you can just work this in to the paragraph above...

File: _posts/development/2020-06-10-events-to-notifications.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+one week ago, that meanwhile changed four times?
+Most likely not, that is why we worked hard to move from an event based flow, to an actual notification
+system that only shows what really matters.
+Instead of producing new notifications records for every single event, we keep notifications
+related to a certain notifiable(can be a comment or request) up-to-date. It will always reflect
+the current state, till the user confirms the acknowledgement by marking it as read.
+
+Badges are indicating the current state of the associated request. A state change of the request will be
+reflected, by the badge. The avatars of users and groups will tell you, who else is involved since the
+last time you read the notification.
```
Comment (hennevogel) [inline _posts/development/2020-06-10-events-to-notifications.md:23]: You should mention the different review bades here

File: _posts/development/2020-06-10-events-to-notifications.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+layout: post
+title: From showing what happend, to showing what matters.
+category: development
+---
+
+It's time again to talk about the notifications. On our journey to a full-fledged notification system,
+we put our emphasis on the content of the notifications itself and keept improving
+several UI parts to increase productivity and user experience.
+
+Until now, a notification just reflects the occurrence of an event in which the notified user is involved.
```
Comment (hennevogel) [inline _posts/development/2020-06-10-events-to-notifications.md:11]:
```suggestion
Until now, a notification just reflects the occurrence of an event in which you are involved.
```

File: _posts/development/2020-06-10-events-to-notifications.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+This can be a response to a comment, a open request that got accepted or a review that
+is wanted from you. The problem here, do you still care about the state of your request from
```
Comment (hennevogel) [inline _posts/development/2020-06-10-events-to-notifications.md:13]:
```suggestion
is wanted from you. The problem with showing a chronological list of events, do you still care about the state of your request from
```

File: _posts/development/2020-06-10-events-to-notifications.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+one week ago, that meanwhile changed four times?
+Most likely not, that is why we worked hard to move from an event based flow, to an actual notification
+system that only shows what really matters.
+Instead of producing new notifications records for every single event, we keep notifications
+related to a certain notifiable(can be a comment or request) up-to-date. It will always reflect
+the current state, till the user confirms the acknowledgement by marking it as read.
+
+Badges are indicating the current state of the associated request. A state change of the request will be
```
Comment (hennevogel) [inline _posts/development/2020-06-10-events-to-notifications.md:21]:
```suggestion
Badges are indicating the current state (e.g. new, superseded or accepted etc.) of the associated request. A state change of the request will be
```

File: _posts/development/2020-06-10-events-to-notifications.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+reflected, by the badge. The avatars of users and groups will tell you, who else is involved since the
+last time you read the notification.
+<img src="/images/posts/sprint-report-79/notification_avatars_and_state.png" alt="Notification avatars and state"/>
+
+For those who wonder where the review wanted notifications remain. This is now unified with the request
+notifications. By showing the latest state, it will reveal when a review is need by you or someone else.
+
+On the UI side of things, there were several improvements that rounds out existing components and
```
Comment (hennevogel) [inline _posts/development/2020-06-10-events-to-notifications.md:29]:
```suggestion
On the design side of things, there were several improvements that rounds out existing components and
```

File: _posts/development/2020-06-10-events-to-notifications.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+filter selectors, giving you a hint at first glance, which kind of notifications are pending.
+
+<img src="/images/posts/sprint-report-79/notification_filter_counter.png" alt="Notification filter counter"/>
+
+Marking a notification as read won't cause full page reload's from now on. We only refresh the parts that needs
+to be refreshed. This saves unnecessary loading times and leads to a better overall experience while scrolling
+through the items.
+
+Mobile devices benfit from a sticky filter section. No need to always scroll back to the top of the page
+in order to apply another filter. The filter section will simply follow, while strolling through your notifications.
```
Comment (hennevogel) [inline _posts/development/2020-06-10-events-to-notifications.md:41]: Both the round trip and the sticky filter are good for smaller viewports. I wouldn't say "Mobile", maybe something like `smaller screen sizes`. 

Also, you forgot about the nice change from @quatran about displaying the filtered project name.

Comment (krauselukas) [reply _posts/development/2020-06-10-events-to-notifications.md:41]: ah yes I missed that

File: _posts/development/2020-06-10-events-to-notifications.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: From showing what happend, to showing what matters.
+category: development
+---
+
+It's time again to talk about the notifications. On our journey to a full-fledged notification system,
+we put our emphasis on the content of the notifications itself and keept improving
```
Comment (danidoni) [inline _posts/development/2020-06-10-events-to-notifications.md:8]:
```suggestion
we put our emphasis on the content of the notifications itself and kept improving
```

File: _posts/development/2020-06-10-events-to-notifications.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+The counters that previously got introduced on the projects filter are now shown on all remaining
+filter selectors, giving you a hint at first glance, which kind of notifications are pending.
+
+<img src="/images/posts/sprint-report-79/notification_filter_counter.png" alt="Notification filter counter"/>
+
+Marking a notification as read won't cause full page reload's from now on. We only refresh the parts that needs
+to be refreshed. This saves unnecessary loading times and leads to a better overall experience while scrolling
+through the items.
+
+Mobile devices benfit from a sticky filter section. No need to always scroll back to the top of the page
```
Comment (danidoni) [inline _posts/development/2020-06-10-events-to-notifications.md:40]:
```suggestion
Mobile devices benefit from a sticky filter section. No need to always scroll back to the top of the page
```

File: _posts/development/2020-06-10-events-to-notifications.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+in order to apply another filter. The filter section will simply follow, while strolling through your notifications.
```
Comment (danidoni) [inline _posts/development/2020-06-10-events-to-notifications.md:41]: `scrolling` or `strolling` like in [stroll](https://www.dictionary.com/browse/strolling)?

**Comment by danidoni** [comment]
Good work! I wish I could write like this! :books:

File: _posts/development/2020-06-10-events-to-notifications.md (hunk 10 above)
Comment (krauselukas) [reply _posts/development/2020-06-10-events-to-notifications.md:41]: `strolling` is intended in this case :)

Comment (danidoni) [reply _posts/development/2020-06-10-events-to-notifications.md:41]: Alright :+1: !

**Pull request merged**